### PR TITLE
Lwen branch 0

### PR DESCRIPTION
**Pull request by lwen**
my last commit should be "remove commented code and debug left-over"

### CODE REVIEW

File: BRAINSSurfaceTools/BRAINSSurfaceGeneration/BRAINSSurfaceGeneration.cxx [hunk 1]
```diff
@@ -239,7 +240,8 @@ int main( int argc, char **argv )
   }
 
   //write the output surface
-  typedef itk::QuadEdgeMeshScalarDataVTKPolyDataWriter< MeshType >  WriterType;
+  //typedef itk::QuadEdgeMeshScalarDataVTKPolyDataWriter< MeshType >  WriterType;
```
Comment (Chaircrusher) [inline BRAINSSurfaceTools/BRAINSSurfaceGeneration/BRAINSSurfaceGeneration.cxx]: Don't leave commented-out code in your source files. We can see what used to be there with git diff, if necessary. Commented-out code just makes it harder to read.

File: BRAINSSurfaceTools/BRAINSSurfaceRegister/AverageScalarsByResampling/AverageScalarsByResampling.cxx [hunk 2]
```diff
@@ -155,7 +155,7 @@ int main( int argc, char * argv [] )
 	  resampleFilter->SetInput (referenceMesh);
 	  resampleFilter->SetReferenceMesh (deformedFixedMesh);
 	  resampleFilter->Update();
-
+std::cout<<"stop here!!!!"<<std::endl;
```
Comment (Chaircrusher) [inline BRAINSSurfaceTools/BRAINSSurfaceRegister/AverageScalarsByResampling/AverageScalarsByResampling.cxx]: Is this left-over debug code?

File: BRAINSSurfaceTools/BRAINSSurfaceGeneration/BRAINSSurfaceGeneration.cxx (hunk 1 above)
Comment (lwen) [inline BRAINSSurfaceTools/BRAINSSurfaceGeneration/BRAINSSurfaceGeneration.cxx]: I made the change. can you see the update?

File: BRAINSSurfaceTools/BRAINSSurfaceRegister/AverageScalarsByResampling/AverageScalarsByResampling.cxx (hunk 2 above)
Comment (lwen) [inline BRAINSSurfaceTools/BRAINSSurfaceRegister/AverageScalarsByResampling/AverageScalarsByResampling.cxx]: I made the change. can you see the update?

Comment (Chaircrusher) [inline BRAINSSurfaceTools/BRAINSSurfaceRegister/AverageScalarsByResampling/AverageScalarsByResampling.cxx]: I think you need to push the branch to your fork, and then submit a new

## pull request, and delete your old pull request.  Is this correct, hans?

Kent Williams norman-k-williams@uiowa.edu

On 6/26/12 12:12 PM, "lwen"
<reply+p-1052885-2fb15493aa24705f008994f249645cb053eabf00-482621@reply.gith
ub.com> wrote:

> > @@ -155,7 +155,7 @@ int main( int argc, char \* argv [] )
> >        resampleFilter->SetInput (referenceMesh);
> >        resampleFilter->SetReferenceMesh (deformedFixedMesh);
> > 
> > ##        resampleFilter->Update();
> > 
> > +std::cout<<"stop here!!!!"<<std::endl;
> 
> I made the change. can you see the update?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/BRAINSia/BRAINSStandAlone/pull/87/files#r1052885

---

Notice: This UI Health Care e-mail (including attachments) is covered by the Electronic Communications Privacy Act, 18 U.S.C. 2510-2521, is confidential and may be legally privileged.  If you are not the intended recipient, you are hereby notified that any retention, dissemination, distribution, or copying of this communication is strictly prohibited.  Please reply to the sender that you have received the message in error, then delete it.  Thank you.

---

Comment (lwen) [inline BRAINSSurfaceTools/BRAINSSurfaceRegister/AverageScalarsByResampling/AverageScalarsByResampling.cxx]: I have pushed my change to the github repository and adjust the commit.

Wen

**Pull request merged**